### DISCUSSION TITLE
My life has been destroyed by cyber attacks for 28 days

### DISCUSSION THREAD

**Discussion by allpropropertiesc** [community/community#126609] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Guys i'm not looking for revenge i'm looking for answers and direction the last 28 days my life has been completely destroyed turning my family upside down someone is attacked every part of my life in the cyber world i need answers i have IP addresses router numbers server numbers i have every documentation i need that's happening for the last 28 days i'm just not text savvy i need someone who is that is really good and that can help me please hit me up privately my email is * litebrite82 removed Personal Identifiable Information

**Comment by LiteBrite82** [comment]
Hi @allpropropertiesc ,

I've removed your private information from [this post](https://github.com/orgs/community/discussions/126609#discussion-6752040) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Reply by LiteBrite82** [reply]
If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this subject.

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic. I'm going to move this over to the "General" category for now. 

Best of luck!

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation, you need to verify it


Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!

**Comment by ebndev** [comment]
I’m sorry that you’re having to go through this, @allpropropertiesc. 

It sounds like you have been a victim of serious harassment and/or abuse. Though we can’t directly help with your situation, you might be able to receive assistance through [Crash Override Network](http://www.crashoverridenetwork.com/) and [RAINN](https://www.rainn.org/).
